setup.py

```diff
@@ -30,7 +30,7 @@
     "pynacl~=1.5",
     "python-dateutil~=2.0",
     "pyyaml~=6.0",
-    "setuptools",
+    "setuptools<70",
```

Comment:
Why not 
```suggestion
    "setuptools<71",
```
This is the one we have been using in the workaround

Reply:
I know. I deviated from that deliberately. From the PR description: "I went for <70 rather than <71 because the identity of the InvalidRequirement exception changed between 69 and 70. This is not as breaking as the issue we were observing with 71, but it might result in some less helpful error messages."

Reply:
ok!